src/analysis/special_functions/gamma.lean

```diff
@@ -75,13 +77,17 @@ by simpa only [Gamma_integral, sub_self, rpow_zero, mul_one] using integral_exp_
 end real
 
 namespace complex
+/- Technical note: In defining the Gamma integrand exp (-x) * x ^ (s - 1) for s complex, we have to
+make a choice between ↑(real.exp (-x)), complex.exp (↑(-x)), and complex.exp (-↑x), all of which are
+equal but not definitionally so. We use the first of these throughout. -/
+
 
 /-- The integral defining the Γ function converges for complex `s` with `1 ≤ re s`.
 
 This is proved by reduction to the real case. The bound is not optimal, but the optimal bound
 (convergence for `0 < re s`) is hard to establish with the results currently in the library. -/
 lemma Gamma_integral_convergent {s : ℂ} (hs : 1 ≤ s.re) :
-  integrable_on (λ x:ℝ, real.exp (-x) * x ^ (s - 1) : ℝ → ℂ) (Ioi 0) :=
+  integrable_on (λ x, (-x).exp * x ^ (s - 1) : ℝ → ℂ) (Ioi 0) :=
```

Comment:
I personally prefer `exp x` over `x.exp`. If the pretty printer is using dot-notation, we should flip some switch so that it will write `exp` as a prefix. I forgot how to do that. @eric-wieser do you know how to do that?

Reply:
The use of dot-notation was to avoid having to explicitly specify real vs. complex -- if you just write `exp (-x)` it will read that as `complex.exp (-↑x)`.

Reply:
`@[pp_nodot]` or similar is the incantation

Reply:
@loefflerd fair point. I think that's a reasonable argument for keeping the dot-notation.